portfolio/src/main/webapp/index.html

```diff
@@ -2,18 +2,65 @@
 <html>
   <head>
     <meta charset="UTF-8">
-    <title>My Portfolio</title>
+    <title>Katherine Yu</title>
     <link rel="stylesheet" href="style.css">
     <script src="script.js"></script>
   </head>
   <body>
     <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click <a href="/hello">here</a> to see content created by server code.</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
-      <div id="greeting-container"></div>
+      <h1 style="text-align:center">Katherine Yu</h1>
+      <div style="text-align:center">
+        <a href="https://www.linkedin.com/in/katherine-y-yu/" target="_blank"><img src="https://static-exp1.licdn.com/sc/h/akt4ae504epesldzj74dzred8" alt="LinkedIn" style="width:30px;vertical-align:middle;margin:0px 10px"></a>
+        <a href="https://github.com/katherine-yu" target="_blank"><img src="https://github.githubassets.com/favicons/favicon.svg" alt="GitHub" style="width:30px;vertical-align:middle;margin:0px 10px"></a>
```

Comment:
While there is nothing wrong with embedding style within the HTML, sometimes it is useful to keep all of your styling logic in your CSS file especially in this case since you have a common style in between these images and you can reuse it. Say, if you decide to add another profile like Facebook, you don't have to copy + paste the same style logic, you can just add a class which will automatically apply the same style.